pkgs/development/tools/hotdoc/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, python3Packages
+, pkg-config
+, flex
+, cmake
+, glib
+, json-glib
+, libxml2
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "hotdoc";
+  version = "0.12.2";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0nypvcqp4b0qfps35f7mz3d90zjh0v5wawpd06ra28q92nxblx17";
+  };
+
+  # https://hotdoc.github.io/installing.html#build-options
+  HOTDOC_BUILD_C_EXTENSION = "enabled";
```

Comment:
Could you reorder the expression as in https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Done.